src/main/java/seedu/kitchenhelper/ui/Ui.java

```diff
@@ -71,11 +74,14 @@ public void validUserChoice(String userChoice) {
             System.out.println(MESSAGE_FOR_NORMAL_MODE);
         } else if (userChoice.trim().equals("2")) {
             System.out.println(MESSAGE_FOR_RESTORE_MODE);
+        } else if (userChoice.equalsIgnoreCase(ExitCommand.COMMAND_WORD)) {
```

Comment:
Would this be execute if `exit   ` was to be entered by the user?

Reply:
Nope it wont execute exit command, it shows message to ask users to enter a loading option first then re-enter exit

Reply:
@zi-hui  what i meant was, you never trim the user input. So when user type `exit[][][]` where [] is the spaces. will this statement be executed?

Reply:
ok i made the change, thankss!!